brainscore/benchmarks/baker2022.py

```diff
@@ -0,0 +1,153 @@
+By contrast, DNNs were more successful when the parts of the silhouettes were rearranged, a manipulation that 
+kept many local shape features but disrupted the overall shape.  Humans show the opposite pattern. "
+
+In other words:
+
+Local Disruption = Humans Ok, Models Bad
+Global Disruption = Humans Bad, Models Ok
+
+Baker 2022 tests for pictures that disrupt global features, but keep local features. That is, models should 
+do much more poorly on these images then humans. 
+
+'''
+
+BIBTEX = """@article{BAKER2022104913,
+                title = {Deep learning models fail to capture the configural nature of human shape perception},
+                journal = {iScience},
+                volume = {25},
+                number = {9},
+                pages = {104913},
+                year = {2022},
+                issn = {2589-0042},
+                doi = {https://doi.org/10.1016/j.isci.2022.104913},
+                url = {https://www.sciencedirect.com/science/article/pii/S2589004222011853},
+                author = {Nicholas Baker and James H. Elder},
+                keywords = {Biological sciences, Neuroscience, Sensory neuroscience},
+                abstract = {Summary
+                A hallmark of human object perception is sensitivity to the holistic configuration of the local shape features of an object. Deep convolutional neural networks (DCNNs) are currently the dominant models for object recognition processing in the visual cortex, but do they capture this configural sensitivity? To answer this question, we employed a dataset of animal silhouettes and created a variant of this dataset that disrupts the configuration of each object while preserving local features. While human performance was impacted by this manipulation, DCNN performance was not, indicating insensitivity to object configuration. Modifications to training and architecture to make networks more brain-like did not lead to configural processing, and none of the networks were able to accurately predict trial-by-trial human object judgements. We speculate that to match human configural sensitivity, networks must be trained to solve a broader range of object tasks beyond category recognition.}
+        }"""
+
+DATASETS = ['normal', 'inverted']
+
+# create functions so that users can import individual benchmarks as e.g. Baker2022wholeAboveChanceAgreement
+for dataset in DATASETS:
+    # normal experiment
+    identifier = f"Baker2022{dataset.replace('_', '')}AccuracyDelta"
+    globals()[identifier] = lambda dataset=dataset: _Baker2022AccuracyDelta(dataset)
+
+    # # inverted experiment
+    # identifier = f"Baker2022_Inverted{dataset.replace('_', '')}AccuracyDelta"
+    # globals()[identifier] = lambda dataset=dataset: _Baker2022InvertedAccuracyDelta(dataset)
+
+
+class _Baker2022AccuracyDelta(BenchmarkBase):
+
+    def __init__(self, dataset, image_types):
```

Comment:
what is `image_types`? Suggest adding some doc and type hints (e.g. `dataset: str, image_types: List[str]`)